matlab_simulation/code v1.0/ME597-7-NonlinearSteering/angleWrap.m

```diff
@@ -0,0 +1,17 @@
+%% angleWrap
+% A function that receives an angle and binds it between [-pi, pi].
+%
+% Input:
+% angle - Any arbitray angle measurement in radians
+%
+% Output:
+% angle - The same angle except numerically bound from -pi to pi
+function [angle] = angleWrap(angle)
+
+while ( angle < -pi || angle > pi)
+    if (angle < -pi)
+        angle = angle + 2*pi;
+    else
+        angle = angle - 2*pi;
+    end
+end
```

Comment:
While it is indeed looks nicer with 1 line, the reasoning isn't as clear as. This could be explained with a comment obviously, nice hack Stan.